shared/xccdf/services/base.xml

```diff
@@ -117,7 +117,7 @@ multiple processors.
 <ocil><systemd-check-macro enable="false" service="irqbalance" /></ocil>
 <rationale>In an environment with multiple processors (now common), the irqbalance service
 provides potential speedups for handling interrupt requests.</rationale>
-<platform idref="cpe:/a:machine"/>
+<machine />
```

Comment:
How about `environment-restriction`? I have made updates using this name...
I like the idea to use this element to filter for multiple environments.